### ISSUE TITLE
Meteor Fatal crash upon adding package

### ISSUE THREAD

**Issue by bvanderdrift** [hwillson/meteor-stub-collections#22]
I am currently setting up a test suite for an application that is being developed. Upon running:

`meteor add hwillson:stub-collections`

My meteor instance crashes with the error.

`TypeError: Path must be a string. Received undefined`

I don't feel comfortable sharing the full trace, there is a lot of system/application information in there.

.meteor/packages:

```
# Meteor packages used by this project, one per line.
# Check this file (and the other files in this directory) into your repository.
#
# 'meteor add' and 'meteor remove' will edit this file for you,
# but you can also edit it by hand.

meteor-base@1.2.0             # Packages every Meteor app needs to have
mobile-experience@1.0.5       # Packages for a great mobile UX
mongo@1.3.0                   # The database Meteor supports right now
reactive-var@1.0.11            # Reactive variable for tracker
tracker@1.1.3                 # Meteor's client-side reactive programming library

standard-minifier-css@1.3.5   # CSS minifier run for production mode
standard-minifier-js@2.2.0    # JS minifier run for production mode
es5-shim@4.6.15                # ECMAScript 5 compatibility for older browsers
ecmascript@0.9.0              # Enable ECMAScript2015+ syntax in app code
shell-server@0.3.0            # Server-side component of the `meteor shell` command

static-html
react-meteor-data
check

meteortesting:mocha
dburles:factory
hwillson:stub-collections
```
package.json:

```
{
  "name": "--",
  "private": true,
  "scripts": {
    "start": "meteor run"
  },
  "dependencies": {
    "babel-runtime": "^6.26.0",
    "meteor-node-stubs": "^0.3.2",
    "react": "^16.1.0",
    "react-dom": "^16.1.0",
    "react-mounter": "^1.2.0"
  },
  "devDependencies": {
    "chai": "^4.1.2",
    "faker": "^4.1.0"
  }
}
```

**Comment by hwillson** [comment]
Hmm - thanks for reporting this @bvanderdrift. I published a new version yesterday, and it looks like something went wrong. I'm working on it now and will have a new version published shortly. Thanks and sorry for the inconvenience!

**Comment by hwillson** [comment]
This should be fixed now - please update to version 1.0.6. The problem came from the fact that when I deployed I forgot to remove the package's local `node_modules` directory (see https://github.com/meteor/meteor/issues/9193). Let me know if you notice any other issues @bvanderdrift - thanks!

**Comment by bvanderdrift** [comment]
Seems to work like a charm! Thanks @hwillson for the very quick response.
